src/services/user.service.ts

```diff
@@ -1,4 +1,5 @@
 import { User } from "src/types/types";
+import "regenerator-runtime"
```

Comment:
Any reason this needed to be in this specific file?

Reply:
Yes, during the docker build with `build:dev` when you perform a `docker run -it -d -p 8080:8080 shell:latest` the web application loads however it makes reference to a react dom error.  Like in your demo, all `remoteEntry.js` would be loaded from remote-1, remote-2 container the port exposure would by 3001:8080, 3002:8080 according to the webpack proxy of `npm run start`

```bash
react-dom.development.js:22834  Uncaught ReferenceError: regeneratorRuntime is not defined
    at _getUser (user.service.ts:19:46)
    at getUser (user.service.ts:16:19)
    at eval (App.tsx:46:68)
    at commitHookEffectListMount (react-dom.development.js:23145:26)
    at commitPassiveMountOnFiber (react-dom.development.js:24921:13)
    at commitPassiveMountEffects_complete (react-dom.development.js:24886:9)
    at commitPassiveMountEffects_begin (react-dom.development.js:24873:7)
    at commitPassiveMountEffects (react-dom.development.js:24861:3)
    at flushPassiveEffectsImpl (react-dom.development.js:27034:3)
    at flushPassiveEffects (react-dom.development.js:26979:14)
```

```bash
react-dom.development.js:18687  The above error occurred in the <App> component:

    at App (webpack://shell/./src/App.tsx?:37:66)
    at Router (webpack://shell/./node_modules/react-router/dist/index.js?:1057:15)
    at BrowserRouter (webpack://shell/./node_modules/react-router-dom/dist/index.js?:333:5)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
```